packages/ncat.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+
+class Ncat < Package
+  description 'Ncat - Netcat for the 21st Century - a modern netcat with extra features from the makers of nmap.'
+  homepage 'https://nmap.org/ncat'
+  version '7.60'
+  source_url 'https://nmap.org/dist/nmap-7.60.tar.bz2'
+  source_sha256 'a8796ecc4fa6c38aad6139d9515dc8113023a82e9d787e5a5fb5fa1b05516f21'
+
+  binary_url ({
+    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ncat-7.60-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    x86_64: '5660ea321436f1b9642d318a77e9b0b5c379825ac0611daaf5f82dd794753598',
+  })
+
+  depends_on 'buildessential' => :build
+  depends_on 'filecmd' => :build   #configure uses file 
+
+  def self.build
+    #fixup "/usr/bin/file" -> "file" in the configure script
+    
+    system "sed -i s#/usr/bin/file##{CREW_PREFIX}/bin/file#g libdnet-stripped/configure"
+
+    #without-zenmap in configure removes openssl dependency
+    system "./configure --without-zenmap --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
```

Comment:
This looks great.

Reply:
Ill send those other 3 PRs this evening.

Reply:
Can you please post the output from `crew files ncat` when you get a chance?  I'm curious which files this package installs.

Reply:
I must have an old enough 'crew' binary that I dont have the files command.... but I did a 'find /usr/local' before and after - and this is the (cleaned up) diff of the output:

> /usr/local/etc/crew/meta/ncat.directorylist
> /usr/local/etc/crew/meta/ncat.filelist
> /usr/local/bin/ncat
> /usr/local/share/man/man1/ncat.1
> /usr/local/share/ncat
> /usr/local/share/ncat/ca-bundle.crt

Reply:
OK great.  So we can drop the `--libdir=#{CREW_LIB_PREFIX}` configure option then.  BTW, if you `cat /usr/local/etc/crew/meta/ncat.filelist`, this essentially does the same thing as `crew files ncat`.  If you execute `crew update` followed by `crew upgrade`, you can get the latest updated versions of all packages including the crew command.

Reply:
you want me to remove the --libdir option?  it doesn't hurt even though the package doesn't install libraries...

Reply:
Yes, although it doesn't hurt to leave it, it would be better to remove it so we can tell just by looking at the configure options that we don't have any libraries.